### ISSUE TITLE
Error popup continuously when a node is down

### ISSUE THREAD

**Issue by IoTThinks** [emqx/emqx#10034]
### What happened?

Seem have the same issue on Ubuntu with emqx 5.0.18
NAME="Ubuntu"
VERSION="20.04.5 LTS (Focal Fossa)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 20.04.5 LTS"
VERSION_ID="20.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=focal
UBUNTU_CODENAME=focal

When one node is down, the alert appears on active node
![image](https://user-images.githubusercontent.com/29994971/221567092-f9259637-b463-412d-a844-30352577e3fd.png)

This is on active node.
sudo emqx_ctl broker
sysdescr  : EMQX
version   : 5.0.18
datetime  : 2023-02-27T19:45:06.060786647+07:00
uptime    : 3 hours, 4 minutes, 49 seconds

The popup is gone when the down node is up.

### What did you expect to happen?

To show information for the remaining active node.

### How can we reproduce it (as minimally and precisely as possible)?

Two nodes in the same cluster.
Turn off 1 node and the error will popup continuously.
We can not see the information of the remaining active node.

### EMQX version

<details>

```console
$ ./bin/emqx_ctl broker
sysdescr  : EMQX
version   : 5.0.18
datetime  : 2023-02-27T19:45:06.060786647+07:00
uptime    : 3 hours, 4 minutes, 49 seconds

```

</details>

### OS version

<details>

```console
# On Linux:
$ cat /etc/os-release
NAME="Ubuntu"
VERSION="20.04.5 LTS (Focal Fossa)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 20.04.5 LTS"
VERSION_ID="20.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=focal
UBUNTU_CODENAME=focal

$ uname -a
Linux ip-10-0-6-161 5.15.0-1030-aws #34~20.04.1-Ubuntu SMP Tue Jan 24 15:16:46 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux

</details>


### Log files

<details>

</details>

**Comment by IoTThinks** [comment]
It is supposed to fix in v5.0.16. https://github.com/emqx/emqx/issues/9197
However, I still have the issue in v5.0.18.

**Comment by IoTThinks** [comment]
Yah, I can help to test when the fix is released.
Thanks a lot.

**Comment by zmstone** [comment]
@IoTThinks the plan is to cut v5.0.19 today.

**Comment by zmstone** [comment]
will be included in v5.0.19, which is being tagged now.

**Comment by IoTThinks** [comment]
Tested. The fix is working in v5.0.19.
Thanks a lot. :D

![image](https://user-images.githubusercontent.com/29994971/222176275-477dead6-e37c-4dc5-b40c-c8e49bd36d00.png)
